README.md

```diff
@@ -14,60 +14,129 @@ Get the development version from github with:
 devtools::install_github("lionel-/fontquiver")
 ```
 
-## Usage
 
-### Get installed font
+## Usage
 
-Get the path to an installed font `foo` with the `font_foo()` getters:
+### Fonts installed in the R library
+
+fontquiver is an interface to fonts installed in the R library. It
+provides convenient and structured access, for instance, to the
+Bitstream Vera font family installed with the
+[fontBitstreamVera](https://cran.r-project.org/package=fontBitstreamVera)
+package, or the Liberation family that comes with
+[fontLiberation](https://cran.r-project.org/package=fontLiberation).
+The fonts too heavy to be distributed on CRAN can be accessed by
+fontquiver through Github-only packages such as
+[fontDejaVu](https://github.com/lionel-/fontDejaVu). These collections
+of font are referred to as `fontset` in fontquiver. Call
```

Comment:
Need an example of a fontset, e.g. Each package bundles a set of fonts (or `fontset`) which typically include bold and italic variations, but may also include...